### ISSUE TITLE
Problems in writing depletion chain file

### ISSUE THREAD

**Issue by AI-Pranto** [openmc-dev/openmc#1592]
I have encountered the following problem in the current development branch while creating a depletion chain file using the ENDF-8 library 

```
File "/home/pranto/.local/lib/python3.6/site-packages/openmc/deplete/chain.py", line 433, in export_to_xml
    root_elem.append(nuclide.to_xml_element()
File "/home/pranto/.local/lib/python3.6/site-packages/openmc/deplete/nuclide.py", line 247, in to_xml_element
    rx_elem.set('target', daughter)
File "src/lxml/etree.pyx", line 816, in lxml.etree._Element.set
File "src/lxml/apihelpers.pxi", line 593, in lxml.etree._setAttributeValue
File "src/lxml/apihelpers.pxi", line 1538, in lxml.etree._utf8
TypeError: Argument must be bytes or unicode, got 'int'
```

**Comment by paulromano** [comment]
Yes, I noticed this too and have a fix for it as part of #1586, which should be merged shortly.

**Comment by AI-Pranto** [comment]
Thanks for the fix @paulromano
